DevCycle.podspec

```diff
@@ -18,4 +18,6 @@ Pod::Spec.new do |spec|
   spec.source_files  = "DevCycle/**/*.{h,m,swift}"
   spec.swift_version = '5.0'
 
+  spec.dependency 'LDSwiftEventSource', '~> 3.0'
```

Comment:
also gotta make sure people that the library can be bundled with the framework even if DevCycle is installed with Swift Package Manager and Carthage